### DISCUSSION TITLE
Using multiple NuxtPages independent of each other

### DISCUSSION THREAD

**Discussion by galexrt** [nuxt/nuxt#26540] [Questions]
I'm trying to figure out how to have multiple NuxtPages in an application like "browser" tabs.

I have several pages defined and working, e.g., `/about`, `/map`, `/settings`, etc.
I want a user to be able to have multiple independent tabs open, e.g., if tab 1 is active and the user clicks on a link in the sidebar or in the tab, it would navigate in that tab. Tab 2, 3, and so on would be unaffected by these route changes until they are activated and the user navigates to a different page.
I got as far as having tabs using HeadlessUI with `:unmount="false"` so the NuxtPage wouldn't be "reset" on tab change, but it would either display the same page or "nothing" because I wasn't able to set the route via the ref/`route` prop.

I have seen projects like https://bhuh12.github.io/vue-router-tab/, but these don't seem up-to-date and work for Vue 3/ Nuxt 3 anymore from playing around with them.

I think that I might need a way to create a router per tab or have a way to set the NuxtPage route programmatically, but I haven't been able to do it yet.

**Comment by carsonbird** [comment]
No answer, but I have the same question. 😬
